npm audit fails

We are trying to run `npm audit` on our project and it fails.
After some researching we found that it failed  because of the following line in your package.json...

```
  "dependencies": {
    "flat": "github:lenchvolodymyr/flat#ffe77ef"
}
```

Why are you targeting a specific commit of that package?
Can this be fixed?

This is the error we got...

```
➜ npm audit fix --registry=https://registry.npmjs.org
npm ERR! code ENOAUDIT
npm ERR! audit Your configured registry (https://registry.npmjs.org/) may not support audit requests, or the audit endpoint may be temporarily unavailable.
npm ERR! audit The server said: Invalid package tree, run  npm install  to rebuild your package-lock.json

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/wolfs/.npm/_logs/2019-11-22T08_10_12_806Z-debug.log
```

Running `npm install` as the errors says does not work..
So after some debugging we found it was the flat dependency that coused the error..

--- a/package.json
+++ b/package.json
@@ -88,7 +88,7 @@
     "@phenomnomnominal/tsquery": "^3.0.0",
     "boxen": "^4.1.0",
     "colorette": "^1.1.0",
-    "flat": "github:lenchvolodymyr/flat#ffe77ef",
+    "flat": "5.0.0",
     "gettext-parser": "^4.0.2",
     "glob": "^7.1.4",
     "mkdirp": "^0.5.1",

